frontend/public/components/pod.jsx

```diff
@@ -22,13 +22,13 @@ kind: Pod
 metadata:
   name: example
   labels:
-    app: redis
+    app: hello-openshift
 spec:
   containers:
     - name: key-value-store
```

Comment:
Missed this last review, but probably should change container name here, too.

Reply:
Fixed

Reply:
Thanks, did you push the change?

Reply:
OK, it's fixed now.